### ISSUE TITLE
its a trojan

### ISSUE THREAD

**Issue by BaggedBandit** [Megant88/Fast-Valorant-Triggerbot#6]
a small issue that includes: 'a severe trojan' detected by anti-virus software's :D

**Comment by Megant88** [comment]
You are insanely r*tarded, this is open source you can literally read the source code on the spot, and the binary has been manually approved by UC Staff, they analyse it, you are insanely embarrassing, i hate people like you

**Comment by BaggedBandit** [comment]
vvvvvEEEry unprofessional tbh, be mad at the virus scanner not me. 
saying that its 'insanely embarrassing' is kind of weird, or maybe the exaggeration is to hide your own imperfections by trying to hurt others. (you should look into that :D ) everything okay at home? :p 
well, sorry for the comment, i did indeed not see that it was open source. and to top it all of, just like you did: i hate people like you :)

**Comment by Megant88** [comment]
I'm not a professional in the first place, so I don't care. But you didnt use your brain, GitHub is an open source platform, you are insanely retarded, stop justifying, use your brain next time instead of leaving stupid comments over the place. I never claimed to be perfect too, what imperfection? It is a false positive scan, that is very common. But you are someone who talks without knowing shit! You are indeed retarded, for assuming something like this. I did not hurt anyone, you are just genuinely braindead, so do me a favor and get TF out of here. Everybody with a few brain cells more is gonna agree with me btw.

**Comment by BaggedBandit** [comment]
u mad? sorry for not being a github nerd.
(being unprofessional is not the same as 'not being a professional')
maybe you are the retarded, braindead, piece of shit.
go do some anal with Elon Musk and tell someone else to get 'TF' out of here.
(ermmm > The term "unprofessional" typically refers to behavior or actions that do not conform to accepted standards of professionalism in a given context)

**Comment by ItsanOriot** [comment]
> u mad? sorry for not being a github nerd. (being unprofessional is not the same as 'not being a professional') maybe you are the retarded, braindead, piece of shit. go do some anal with Elon Musk and tell someone else to get 'TF' out of here. (ermmm > The term "unprofessional" typically refers to behavior or actions that do not conform to accepted standards of professionalism in a given context)

Megant is right. It's common knowledge that pretty much anything cheating related will trigger false positives on anti viruses. Be grateful he's giving u free shit instead of refusing to admit when you're wrong and read the source next time before you start making accusations

**Comment by t-nshi** [comment]
> u mad? sorry for not being a github nerd. (being unprofessional is not the same as 'not being a professional') maybe you are the retarded, braindead, piece of shit. go do some anal with Elon Musk and tell someone else to get 'TF' out of here. (ermmm > The term "unprofessional" typically refers to behavior or actions that do not conform to accepted standards of professionalism in a given context)

dawg ur just digging urself deeper into yourembarrassment hole xd

**Comment by BaggedBandit** [comment]
why do you care tbh?

**Comment by Megant88** [comment]
> why do you care tbh?

because you are genuinely wrong here, stop justifying and admit that you lack some iq cells idiot

**Comment by BaggedBandit** [comment]
yes, ik that I'm in the wrong here but that is no reason for me to take those insults like that.
its just a level of respect for other people that does not exist in you guys, yes, i did call you guys some bad stuff but that was to show that i wont really tolerate that behavior.

(you called me an idiot, and said that i have a lack of braincells, that retarded, that I'm braindead, that it is 'insanely embarrassing' over just me saying that a trojan got detected.) > someone that is not a piece of shit would say something like: (to begin with, you called me retarded!) it happens a lot, no worry's Github is open source and there is no risk... etc.
 (beginning a sentence with calling someone retarded is another level of unprofessionally)

**Comment by Megant88** [comment]
> yes, ik that I'm in the wrong here but that is no reason for me to take those insults like that. its just a level of respect for other people that does not exist in you guys, yes, i did call you guys some bad stuff but that was to show that i wont really tolerate that behavior.
> 
> (you called me an idiot, and said that i have a lack of braincells, that retarded, that I'm braindead, that it is 'insanely embarrassing' over just me saying that a trojan got detected.) > someone that is not a piece of shit would say something like: (to begin with, you called me retarded!) it happens a lot, no worry's Github is open source and there is no risk... etc. (beginning a sentence with calling someone retarded is another level of unprofessionally)

Yes and im not wrong, you lack iq cells, you are indeed an idiot and you are also very retarded, everyone agrees on that here, you did not think straight and instantly left an useless comment, and then you go blame the antivirus, it is obvious that this release is not a trojaner for a billion reasons, again you are a dumb fuck. Im not a professional so I dont care about you and what else I could possibly call you clown. Nobody has respect for people who lack common sense, nobody cares if you tolerate our behavior. You are simply braindead and digging yourself into the rabbit hole. Just get tf out of here, stop justyfing.

**Comment by Jeremyop** [comment]
> yes, ik that I'm in the wrong here but that is no reason for me to take those insults like that. its just a level of respect for other people that does not exist in you guys, yes, i did call you guys some bad stuff but that was to show that i wont really tolerate that behavior.
> 
> (you called me an idiot, and said that i have a lack of braincells, that retarded, that I'm braindead, that it is 'insanely embarrassing' over just me saying that a trojan got detected.) > someone that is not a piece of shit would say something like: (to begin with, you called me retarded!) it happens a lot, no worry's Github is open source and there is no risk... etc. (beginning a sentence with calling someone retarded is another level of unprofessionally)

Imagine not checking the source code and calling it a virus

**Comment by swAzEw** [comment]
> yes, ik that I'm in the wrong here but that is no reason for me to take those insults like that. its just a level of respect for other people that does not exist in you guys, yes, i did call you guys some bad stuff but that was to show that i wont really tolerate that behavior.
> 
> (you called me an idiot, and said that i have a lack of braincells, that retarded, that I'm braindead, that it is 'insanely embarrassing' over just me saying that a trojan got detected.) > someone that is not a piece of shit would say something like: (to begin with, you called me retarded!) it happens a lot, no worry's Github is open source and there is no risk... etc. (beginning a sentence with calling someone retarded is another level of unprofessionally)

So why open a issues for dogshit ? You are an idiot.

**Comment by BaggedBandit** [comment]
imagine being French

**Comment by swAzEw** [comment]
I would rather be French than be stupid like you...

**Comment by Megant88** [comment]
> imagine being French

Everyone here has told you now that you are a retard, I do not know these guys but this just proves my point that you are genuinely a dumbass and everyone excepts you seems to see it, so would you piss of now? Congrats to the residency you gained yourself on the hall of shame and now fuck off retard ^

**Comment by lyu0131** [comment]
> u mad? sorry for not being a github nerd. (being unprofessional is not the same as 'not being a professional') maybe you are the retarded, braindead, piece of shit. go do some anal with Elon Musk and tell someone else to get 'TF' out of here. (ermmm > The term "unprofessional" typically refers to behavior or actions that do not conform to accepted standards of professionalism in a given context)

man you really are the worst of the worst....

**Comment by BaggedBandit** [comment]
hey man.. sorry for saying that shit back then, i was new to the coding community and i did not know what i was doing. I said those those things to you because i thought it was unfair to be called slurs but tbh I asked for it. 
Hope you have a good life going forwards.

**Comment by Megant88** [comment]
> hey man.. sorry for saying that shit back then, i was new to the coding community and i did not know what i was doing. I said those those things to you because i thought it was unfair to be called slurs but tbh I asked for it. Hope you have a good life going forwards.

hey man, no worries, no bad blood here, what you said back there was just really stupid and how you reacted made it even worse, and yes never take insults, even if you are in the wrong, but always apologize once you realize you are wrong. And I should have not insulted you straight away and maybe explained it in peace, but then again you asked for it. Whatever, no bad blood, have a nice life friend and never be too shy to learn :)
